.github/workflows/build-and-test.yaml

```diff
@@ -280,7 +284,7 @@ jobs:
             > component-descriptor.yaml
 
           if ${{ inputs.release || false }}; then
-            commit_digest=${{ steps.releasecommit.outputs.release_commit_digest || '' }}
+            commit_digest=${{ needs.version.outputs.release-commit-digest || '' }}
```

Comment:
In which case is the `|| ''` required?

Reply:
good catch! it is in our workflow not required at all, considering we always create a crelease-commit in release-case.